packages/manager/modules/core/src/redirection/redirection.constants.js

```diff
@@ -79,6 +81,7 @@ export default {
       },
     },
     help: 'https://help.ovhcloud.com',
+    order: `${managerRoot.CA}/manager/${URI.order}`,
```

Comment:
Correct me if I'm wrong, isn't supposed to be:

```suggestion
    order: `${managerRoot.CA}/manager/dedicated/${URI.order}`,
```

Reply:
@antleblanc i don't know, currently dedicated CA in on /manager, is it planned to be moved ?

Reply:
From my point of view I would answer **Yes**.

cc @marie-j to confirm.

Reply:
@frenautvh This is future URL for hub, so dedicated in CA has to move